jdbc/src/java/org/apache/hive/jdbc/Utils.java

```diff
@@ -99,6 +99,7 @@ public static class JdbcConnectionParams {
     public static final String AUTH_PASSWD = "password";
     public static final String AUTH_KERBEROS_AUTH_TYPE = "kerberosAuthType";
     public static final String AUTH_KERBEROS_AUTH_TYPE_FROM_SUBJECT = "fromSubject";
+    public static final String AUTH_KERBEROS_ENABLE_CANONICAL_HOSTNAME_CHECK = "enableCanonicalHostnameCheck";
```

Comment:
wondering if the config name should imply that this only applies in case of kerberos auth, e.g.:
```
kerberosEnableCanonicalHostnameCheck
```

Reply:
renamed